tools/ports/__init__.py

```diff
@@ -67,13 +67,9 @@ def init_port(name, port):
   validate_port(port)
 
 
-def load_port_by_name(name):
-  port = __import__(name, globals(), level=1, fromlist=[None])
-  init_port(name, port)
-
-
-def load_port_by_path(path):
-  name = os.path.splitext(os.path.basename(path))[0]
```

Comment:
Unrelated to this change, but I'm a little curious about the `sys.modules[module_name] = port` line below.  Is that necessary?  What happens if you remove it?